front/pages/w/[wId]/builder/assistants/create.tsx

```diff
@@ -88,15 +89,22 @@ export default function CreateAssistant({
     tags: TemplateTagCodeType[];
   }>({ templates: [], tags: [] });
 
+  const whiteListedProviders = useMemo(() => {
```

Comment:
nit: useMemo isn't necessary here as the function is trivial
(useMemo avoids recomputations at every render and should be used for expensive functions)